index.html

```diff
@@ -14017,49 +14026,42 @@ <h4 id="PrimaryGlobal">4.3.6. [Global] and [PrimaryGlobal]</h4>
             <li>has one or more <a class="dfnref" href="#dfn-regular-operation">regular operations</a> that all have the same <a class="dfnref" href="#dfn-identifier">identifier</a>,
               and no others.</li>
           </ul>
+
           <p>
-            A <a class="dfnref" href="#dfn-user-object">user object</a>’s
-            <a class="dfnref" href="#dfn-operation">operation</a> is called
-            with a list of IDL argument values <var>idlarg</var><sub>0..<var>n</var>−1</sub> by
-            following the algorithm below.  The <dfn data-export="" id="dfn-callback-this-value">callback this value</dfn>
-            is the value to use as the <span class="esvalue">this</span> value
-            when a <a class="external" href="https://tc39.github.io/ecma262/#sec-iscallable">callable</a>
-            object was supplied as the implementation of a
-            <a class="dfnref" href="#dfn-single-operation-callback-interface">single operation callback interface</a>.
-            By default, <span class="esvalue">undefined</span> is used as the <a class="dfnref" href="#dfn-callback-this-value">callback this value</a>,
-            however this <span class="rfc2119">MAY</span> be overridden by other
-            specifications.
+            To <dfn>call a user object's operation</dfn>, given a <a class="dfnref" href="#idl-interface">callback interface type</a> value <var>value</var>, sometimes-optional operation name <var>opName</var>, list of IDL argument <var>idlarg</var><sub>0..<var>n</var>−1</sub>, and optional <dfn data-export="" id="dfn-callback-this-value">callback this value</dfn> <var>thisArg</var>, perform the following steps.
```

Comment:
"list of IDL argument" either needs an 's' at the end or the restoration of the "values" that went missing.  Preferably the latter.